docs/node-configuration.md

```diff
@@ -347,6 +348,10 @@ where:
 - `MaxWebSocketFeeds` -- the maximum simultaneous event subscriptions number
   for a single client (16 by default). Attemps to create additional subscriptions
   will lead to error.
+- `MempoolSubscriptionsEnabled` - enables or disables mempool_event WebSocket
```

Comment:
denotes whether JSON-RPC websocket subscription for mempool events is enabled. If not set, a call to `subscribe` JSON-RPC handler with `mempool_event` argument will return an error.